recipes/shared-memory-dict/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "shared-memory-dict" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
```

```suggestion
  name: {{ name|lower }}-meta
```
This package will actually not be built, it is just a meta package.